src/App.jsx

```diff
@@ -85,6 +86,7 @@ const App = () => {
   if (!isLoggedIn) {
     return (
       <HashRouter>
+        {!showPlans && <CookieConsentBanner />}
```

Comment:
for what is that condition?

Reply:
If only one server I available for the frontend, we are showing the cookie banner at the sign in already. But when you can select a server we can't show it, because in the Cookie consent banner we link a pricacy policy and therefore we cant decide which banner to show in the selection screen.

Reply:
Then please use the `BASE_URL` variable.

Reply:
Done